assets/src/components/d3/d3ViewsNarrative.js

```diff
@@ -0,0 +1,57 @@
+import { isInRange, isOutOfRange } from '../../util/math'
+
+const createResourcesText = (resourceData, resourceType, gradeSelection, weekRange) => {
+  const unAccessedResources = resourceData.filter(resource => resource.self_access_count === 0)
+  const accessedResources = resourceData.filter(resource => resource.self_access_count === 1)
```

Comment:
```suggestion
  const accessedResources = resourceData.filter(resource => resource.self_access_count >= 1)
```

This is a bug. In cases where the `self_access_count` was greater than 1, no text was included for these resources.